### PR TITLE
html-forms exercise completed.

### PR DESCRIPTION
**Pull request by joseph-w-valdez**
https://i.imgur.com/7FilkYi.png 
https://i.imgur.com/YgIp7PQ.png
https://i.imgur.com/8RRMpAs.png

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <form>
+    <label for="email">Email</label>
```
[the remark this answers is not in the document]
Comment (joseph-w-valdez) [reply html-forms/index.html:12]: it appears to be working for me on the livereload server page. i can only select one of the three (HTML, CSS, and JavaScript)

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      <select id="cool">
+        <option value="">Select an option</option>
+        <option value="">Because it's interesting</option>
+        <option value="">Becacuse it's fun</option>
+        <option value="">Because it's powerful</option>
+      </select>
+    </div>
+    <p></p>
+    <div>
+      <label for="language">Favorite Language (choose one)</label>
```
Comment (codingcodymiller) [inline html-forms/index.html:27]: This shouldn't be a `label` element, it should be a `p` element. Also, do not use empty `p` elements to create space

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    <div>
+      <input type="radio" id="HTML" value="HTML" name="Language">
+      <label for="HTML">HTML</label>
+      <input type="radio" id="CSS" value="CSS" name="Language">
+      <label for="CSS">CSS</label>
+      <input type="radio" id="JavaScript" value="JavaScript" name="Language">
+      <label for="JavaScript">JavaScript</label>
+    </div>
+    <p></p>
+    <div>Would you like to sign up for more information?</div>
```
Comment (codingcodymiller) [inline html-forms/index.html:39]: This should be a `p` element

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: html-forms/index.html (hunk 2 above)
Comment (joseph-w-valdez) [reply html-forms/index.html]: oh, I misunderstood from the conversation earlier. I've made the adjustments and they should be fixed and pushed now.

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**